trp_use_case/model/use_case.py

```diff
@@ -143,6 +143,30 @@ class use_case_collection(models.Model):
     _name = 'use_case.collection'
     _description = 'Set of use cases'
 
+    state = fields.Selection([
+                 ('draft', 'Draft'),
+                 ('open', 'Open'),
+                 ('done', 'Done'),
+                 ], string="Status", help="Collection Status", default="draft", required = True)
+
+    @api.one
+    def draft_statusbar(self):
```

Comment:
check if you're called by a user who is allowed to do this: https://github.com/OCA/OCB/blob/8.0/openerp/models.py#L1365